### ISSUE TITLE
Handle multiple function projects in one workspace

### ISSUE THREAD

**Issue by EricJizbaMSFT** [Microsoft/vscode-azurefunctions#1121]
@EricJizbaMSFT should this still work?

I just tried with the latest VS Code and Azure Functions tools bits, and it does not work.
I always get a 
![image](https://user-images.githubusercontent.com/888718/54996560-ee819000-4fc9-11e9-90d5-09fd1c284f03.png)

```
Error: The func task detection didn't contribute a task for the following configuration:
{
    "type": "func",
    "dependsOn": "build",
    "options": {
        "cwd": "${workspaceFolder}/sqlserver-demo/bin/Debug/netcoreapp2.1"
    },
    "command": "host start",
    "isBackground": true,
    "problemMatcher": "$func-watch"
}
The task will be ignored.
```

Sample can be found here: https://github.com/thinktecture/azure-functions-extensibility

Thanks!

_Originally posted by @ChristianWeyer in https://github.com/Microsoft/vscode-azurefunctions/issues/686#issuecomment-476601118_

**Comment by EricJizbaMSFT** [comment]
@ChristianWeyer that's happening because you have multiple function projects in the same workspace and we apparently don't handle that well. I'll look into fixing, but in the meantime you can either:
1. Delete 'sqlserver-demo-js' folder
1. Open up the `sqlserver-demo` folder directly
1. Change the "func host start" task in your tasks.json to this:
    ```json
    {
      "type": "shell",
      "label": "runFunctionsHost",
      "dependsOn": "build",
      "options": {
        "cwd": "${workspaceFolder}/sqlserver-demo/bin/Debug/netcoreapp2.1"
      },
      "command": "func host start",
      "isBackground": true,
      "problemMatcher": "$func-watch"
    }
    ```
    (This last option only works because you're debugging C#. It would be a little more complicated if you wanted to debug the JavaScript project instead)

Another option would be to use a [multi-root workspace](https://code.visualstudio.com/docs/editor/multi-root-workspaces), but we haven't released a fix for that yet. Once we release the fix, you can actually run/debug both function projects at the same time. See here for more info: https://github.com/Microsoft/vscode-azurefunctions/issues/1081

**Comment by ChristianWeyer** [comment]
Thanks!
Indeed, the next step would be to debug the JS project (sqlserver-demo-js) and step into the C# code in the extensions.
Any hint for that?

**Comment by EricJizbaMSFT** [comment]
@ChristianWeyer we just released an update to the extension (v0.16.0) which had some improvements in this area. See this wiki for how to handle multiple function projects: https://github.com/Microsoft/vscode-azurefunctions/wiki/Multiple-function-projects
